simplivity/resources/virtual_machines.py

```diff
@@ -340,3 +340,20 @@ def power_off(self, timeout=-1):
         self.__refresh()
 
         return self
+
+    def power_on(self, timeout=-1):
+        """Power on virtual machine.
+
+        Args:
+            timeout: Time out for the request in seconds.
+
+        Returns:
+            self: Returns the same object.
```

Comment:
This should return true or false

you don't want the end user to have to replace their existing objects

Reply:
Will update as per suggestion

Reply:
Done